modules/multilanguage/src/main/scala/com/ukg/surge/multilanguage/GenericAsyncAggregateCommandModel.scala

```diff
@@ -42,53 +43,60 @@ class GenericAsyncAggregateCommandModel(bridgeToBusinessApp: BusinessLogicServic
     }
   }
 
-  override def processCommand(aggregate: Option[SurgeState], surgeCommand: SurgeCmd): Future[Seq[SurgeEvent]] = {
-    if (!validIds(aggregate, surgeCommand)) {
-      Future.failed(new Exception("Wrong aggregate ids!"))
-    } else {
-      logger.info(
-        s"Calling command handler of business app via gRPC. Aggregate id: ${surgeCommand.aggregateId}). State defined: ${aggregate.isDefined}. Command payload size: ${surgeCommand.payload.length} (bytes).")
+  private val metric: Metrics = Metrics.globalMetricRegistry
+  private val processCommandMetric: Timer = metric.timer(MetricInfo("ProcessCommand", "Process Command Metric"))
```

Comment:
Can you make the description a bit more descriptive here and scope the metric name a bit as well? Maybe something like `surge.grpc.process-command-timer` for the name.

Reply:
done.